readthedocs/projects/views/private.py

```diff
@@ -783,11 +783,12 @@ class IntegrationCreate(IntegrationMixin, CreateView):
 
     def form_valid(self, form):
         self.object = form.save()
-        attach_webhook(
-            project_pk=self.get_project().pk,
-            user_pk=self.request.user.pk,
-            integration=self.object
-        )
+        if self.object.integration_type != Integration.API_WEBHOOK:
```

Comment:
Do we not have some kind of `has_sync` check here? I feel like this might break in the future as we add new types of Integration.

Reply:
Thanks. I think what you said would be better. :)

Reply:
@ericholscher Updated the PR!